src/components/edit-dl-members.tsx

```diff
@@ -32,36 +34,38 @@ const createDuplicatedMemberAction = (): ChipAction => ({
 	icon: 'AlertCircle'
 });
 
-export const EditDLComponent: FC<EditDLComponentProps> = ({
-	email,
+export const EditDLMembersComponent: FC<EditDLComponentProps> = ({
 	members,
 	totalMembers,
 	onRemoveMember,
-	onAddMembers
+	onAddMembers,
+	loading
 }) => {
 	const [t] = useTranslation();
 	const [contactInputValue, setContactInputValue] = useState<ContactInputValue>([]);
 	const [searchValue, setSearchValue] = useState('');
 
 	const memberItems = useMemo(
 		() =>
-			reduce<string, React.JSX.Element[]>(
-				members,
-				(accumulator, member) => {
-					if (member.includes(searchValue)) {
-						accumulator.push(
-							<MemberListItemComponent
-								email={member}
-								onRemove={(): void => onRemoveMember(member)}
-								key={member}
-							/>
-						);
-					}
-					return accumulator;
-				},
-				[]
-			),
-		[members, onRemoveMember, searchValue]
+			(!loading &&
+				reduce<string, React.JSX.Element[]>(
+					members,
+					(accumulator, member) => {
+						if (member.includes(searchValue)) {
+							accumulator.push(
+								<MemberListItemComponent
+									email={member}
+									onRemove={(): void => onRemoveMember(member)}
+									key={member}
+								/>
+							);
+						}
+						return accumulator;
+					},
+					[]
+				)) ||
+			times(3, (index) => <ShimmedDisplayerListItem key={index} />),
```

Comment:
Can it be usefull to create a ShimmedList component? To keep consistency between all the lists in the module. I believe i have already seen the same code around

Reply:
I can group the times function inside another function or component, or/and I could remove (which I like better) the loading items from here and place them inside the list directly, conditionally rendering them based on the loading state.
Something like
```tsx
<ListV2>
    {loading && <LoadingItems coout={3} />}
    {!loading && memberItems}
</ListV2>
```
What do you think about this?

Reply:
This seems much better!